src/lib.rs

```diff
@@ -13,6 +13,102 @@
 #![recursion_limit = "128"]
 #![type_length_limit = "1572864"]
 
+//! TiKV Client for Rust.
+//!
+//! > Currently this crate is experimental and some portions (eg the Transactional API) are still
+//! > in active development. You're encouraged to use this library for testing and help us find
+//! > problems!
+//!
+//! This crate provides Rust bindings to [TiKV](https://github.com/tikv/tikv), a distributed
+//! transactional Key-Value database written in Rust.
+//!
+//! With this crate you can easily connect to any TiKV deployment, interact with it, and mutate the
+//! data it contains.
+//!
+//! This is an open source (Apache 2) project hosted by the Cloud Native Computing Foundation
+//! (CNCF) and maintained by the TiKV Authors. *We'd love it if you joined us in improving this
+//! project.*
+//!
+//! ## Install
+//!
+//! There are no special requirements to use this. It is a Rust 2018 edition crate supporting
+//! stable and nightly.
+//!
+//! To use this crate in your project, depend on it in the `Cargo.toml` of your Rust project:
+//!
+//! ```toml
+//! [dependencies]
+//! # ...Your other dependencies...
+//! tikv-client = "~0.1"
+//! futures = "0.1" # You'll need this later.
+//! ```
+//!
+//! Then give a `cargo build --package tikv-client` to test building the crate.
+//!
+//! Next, you need to choose the API appropriate for your needs.
+//!
+//! ## Choosing an API
+//!
+//! This crate offers both [**raw**](raw/index.html) and
+//! [**transactional**](transaction/index.html) APIs. You should choose just one for your system.
+//!
+//! The *consequence* of supporting transactions is increased overhead of coordination with `pd`
```

Comment:
```suggestion
//! The *consequence* of supporting transactions is increased overhead of coordination with `PD`
```

Reply:
I'm not sure about this one. 🤔

Reply:
I do see the API docs elsewhere use PD, without ticks or quotes.

Reply:
PD is our official short name for Placement Driver. It is safe to spell out "the placement driver" without the backtick since it is the first appearance here.